packages/config/config/devices/0x0346/outdoor_contact_sensor.json

```diff
@@ -0,0 +1,106 @@
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "B0923BK77S",
+	"description": "Outdoor Contact Sensor",
+	"devices": [
+		{
+			"productType": "0x0c01",
+			"productId": "0x0301"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Heartbeat Interval",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 1,
+			"maxValue": 70,
+			"defaultValue": 70,
+			"unsigned": true
+		},
+		{
+			"#": "2",
+			"label": "Message Retry Attempt Limit",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "Delay Between Retry Attempts",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 60,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "4",
+			"label": "LED Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Always off",
+					"value": 0
+				},
+				{
+					"label": "On when activated",
+					"value": 1
+				},
+				{
+					"label": "On when activated or cleared",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Force Wakeup After Delay",
+			"description": "Device will wakeup once after the configured delay",
```

Comment:
Did you copy this from a different device config? If not, I'd call this "Wakeup Interval" or something like that.

Reply:
Yes, I reused all of the parameter labels from the [Ring Contact Sensor Gen2](https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0346/contact_sensor_gen2.json#L85).

The manual for both has this as the description:

> One shot timer:
> Writing to this parameter prompts the sensor to send a wakeup notification one time after this parameter’s number of seconds. After which it is reset back to 0.

If you'd like I can update both labels in this PR.

Reply:
Ohh that is completely different than what I understood. Please do, you can include it in this PR.

Reply:
What should it be updated to? "Force Wakeup After Delay" sounds like a boolean, so it would make sense to me to change it. Is the text from the manual ("One Shot Timer") clear?

Reply:
Something like _One-Time Wakeup After Delay_ maybe?

Reply:
Sounds good. Pushed a commit with updated text for both devices.